.travis.yml

```diff
@@ -1,13 +1,15 @@
 ---
+sudo: false
 language: ruby
 
+notifications:
+  email: false
+
 rvm:
   - 2.1.9
```

Comment:
Let's support the same ruby version shipping in the AIO package of the Puppet Agent the customer support.  So we can start with Puppet 4, Puppet 5 with MRI 2.1.9 and 2.4.1 and add future MRI versions for Puppet 6 onward.